doc/moveit_cpp/src/moveit_cpp_tutorial.cpp

```diff
@@ -115,10 +114,9 @@ int main(int argc, char** argv)
   {
     // Visualize the start pose in Rviz
     visual_tools.publishAxisLabeled(robot_start_state->getGlobalLinkTransform("panda_link8"), "start_pose");
-    visual_tools.publishText(text_pose, "Start Pose", rvt::WHITE, rvt::XLARGE);
     // Visualize the goal pose in Rviz
     visual_tools.publishAxisLabeled(target_pose1.pose, "target_pose");
-    visual_tools.publishText(text_pose, "Goal Pose", rvt::WHITE, rvt::XLARGE);
+    visual_tools.publishText(text_pose, "setStartStateToCurrentState", rvt::WHITE, rvt::XLARGE);
```

Comment:
Shouldn't this be `Goal pose`?

Reply:
This text is what is displayed above the robot to let the user know what section of the tutorial they are in. We publish a `AxisLabeled` on the poses.